### ISSUE TITLE
Add editing controls to allow people assign aria-label attributes.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#120]
**There is a bug reported by Microsoft accessibility team "‘Try it out’ link text is not descriptive present on home page.";**

User Experience :

If link purpose is not descriptive then screen reader user will not be able to find out the proper information and will be confused and will face difficulties while navigating on the page.

Note: User credentials should NOT be included in the bug.
 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode

4. Enable screen reader and navigate to the various controls of the "Home" page like left nav, bottom details.

 

Actual Result:

‘Try it out’ link text is not descriptive, screen reader only announces ‘Try it out’ link for all ‘Try it Out’ link present on home page. It will be very difficult to screen reader users to find out the purpose of ‘Try it out’ links if many ‘Try it out’ links is present on the page.

 

Expected Result:

‘Try it Out’ links text should be descriptive for all ‘Try it out’ links, screen reader should announce it as ‘Contoso API 1’ try it out link, Contoso API 2 try it out link, etc.

Note:

Similar issue exist with Chrome + JAWS, Chrome + NVDA

MAS Reference:

MAS 2.4.4 - Link Purpose (In Context)

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.
